wsi_deid/web_client/templates/ItemView.pug

```diff
@@ -36,4 +36,17 @@
         input.g-refile-imageid
       button.g-refile-button.btn(class='btn-'+buttons.refile.color, action=button, title=buttons.refile.title) #{buttons.refile.name}
   for button in buttonList
-    button.g-workflow-button.btn(class='btn-'+buttons[button].color, action=button, title=buttons[button].title) #{buttons[button].name}
+    - var isDisabled = button === 'reject' && require_reject_reason
+    button.g-workflow-button.btn(class='btn-'+buttons[button].color, action=button, title=buttons[button].title, disabled=isDisabled) #{buttons[button].name}
+  if buttonList.includes('reject') && rejection_reasons.length > 0
```

Comment:
This should also check if the reject reason is required (and, if not, not show the drop down).

Reply:
Added a check in 5ade89f